tracing-subscriber/src/fmt/span.rs

```diff
@@ -223,6 +229,15 @@ fn id_to_idx(id: &Id) -> usize {
     id.into_u64() as usize - 1
 }
 
+#[inline]
+fn find_last_no_duplicate(stack: &[ContextId]) -> Option<&Id> {
```

Comment:
Nit/Take it or leave it: If the stack was a new type rather than just being a `Vec<ContextId>` (which, if we decide to take the suggestion I gave later on), this would probably make more sense as a method...

Reply:
yes!